src/bentoml/_internal/resource.py

```diff
@@ -244,6 +213,13 @@ def from_system(cls) -> list[int]:
         """
         import pynvml  # type: ignore
 
+        cuda_visible_devices = os.environ.get("CUDA_VISIBLE_DEVICES", None)
+        if cuda_visible_devices is not None:
+            if "," in cuda_visible_devices:
+                return [int(i) for i in cuda_visible_devices.split(",")]
+            else:
```

Comment:
Seems like the spec says comma separated (https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#env-vars); why are we supporting space?